src/Microsoft.Sbom.Api/Executors/Spdx2SerializationStrategy.cs

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.Entities;
+using Microsoft.Sbom.Api.Providers;
+using Microsoft.Sbom.Extensions;
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+/// <summary>
+/// Serialization methods for SPDX 2.2.
+/// </summary>
+public class Spdx2SerializationStrategy : IJsonSerializationStrategy
```

Comment:
For consistency with the rest of the code, should this be named `Spdx22SerializationStrategy` ? (with '22' instead of '2' as the version)

Reply:
Yes that makes sense, I've renamed all these instances to make it consistent.